unittests/test_schedulers.py

```diff
@@ -381,11 +392,12 @@ def test_submit(make_job, exec_ctx):
 
     # Additional scheduler-specific checks
     sched_name = minimal_job.scheduler.registered_name
+
     if sched_name == 'local':
         assert [socket.gethostname()] == minimal_job.nodelist
         assert minimal_job.exitcode == 0
         assert minimal_job.state == 'SUCCESS'
-    elif sched_name == ('slurm', 'squeue', 'pbs', 'torque'):
+    elif sched_name in ('slurm', 'squeue', 'pbs', 'torque'):
```

Comment:
Remove the `squeue` because in that case, neither of the `nodelist`, `exitcode` are retrieved. This is why the CSCS CI fails.